docs/qute/TemplateSupport.md

```diff
@@ -94,6 +94,12 @@ A variable can be declared inside a Qute template (with parameter declaration, o
 
 ![Definition for Java data model](images/QuteTemplateDefinitionForObject.gif)
 
+## User tags support
+
+The [User tags](https://quarkus.io/guides/qute-reference#user_tags) are loaded from `src/main/resources/templates/tags` and uses content of user tag to collect parameters from the object part, uses the existing of `{nested-content}` to provide the best support for completion, definition, validation:
```

Comment:
I think we can reword this part to something like: 
... are loaded from `src/main/resources/templates/tags`. The parameters and presence of `{nested-content}` in the user tag declared in the tag template is used to provide support for completion, definition, and validation in the Qute template:

Reply:
fixed